src/libs/ajax/Cbas.js

```diff
@@ -1,15 +1,66 @@
 import _ from 'lodash/fp';
-import { authOpts, fetchCbas, jsonBody } from 'src/libs/ajax/ajax-common';
+import qs from 'qs';
+import { authOpts, fetchCbas, fetchCromwell, fetchLeo, fetchOk, jsonBody } from 'src/libs/ajax/ajax-common';
+import { getConfig } from 'src/libs/config';
 
 export const Cbas = (signal) => ({
```

Comment:
I won't push hard on this PR for more Typescript since I assume you have a multi-PR plan.  But I will highlight that having good Typescript decorations on the api call contracts (arg types and Promise<ResultType>) is very very valuable.